pycbc/population/scale_injections.py

```diff
@@ -0,0 +1,796 @@
+
+       Returns
+       -------
+       data: dictionary
+           Dictionary containing injection read from the input file
+    """
+    data = {}
+    
+    with h5py.File(hdffile, 'r') as inp:
+        found_index = inp['found_after_vetoes/injection_index'][:]
+
+        for param in _save_params:
+            data[param] = inp['injections/'+param][:]
+        
+        ifar = np.zeros_like(data[_save_params[0]])
+        ifar[found_index] = inp['found_after_vetoes/ifar'][:]
+        
+        data['ifar'] = ifar
+        
+        stat = np.zeros_like(data[_save_params[0]])
+        stat[found_index] = inp['found_after_vetoes/stat'][:]
+        
+        data['stat'] = stat
+
+    return data
+
+def merge_injections(all_inj):
+    """Merge injections across chunks.
+
+       Parameters
+       ----------
+       all_inj: dictionary
+           dictionary containing injections for various chunks
+
+       Returns
+       -------
+       data: dictionary
+           Dictionary containing the merged injections
+    """
+    injs = {}
+    for data in np.append(_save_params, 'found'):
+        injs[data] = np.concatenate([inj[data] for inj in tuple(all_inj.values())])
+   
+    return injs
+        
+def m1m2_to_mcheta(m1, m2):
+    ''' Get chirp mass and eta for m1 and m2
+
+       Parameters
+       ----------
+       m1: array/float
+           First mass of the binary
+       m2: array/float
+           Second mass of the binary
+
+       Returns
+       -------
+       array/float
+           Chirp mass for the m1, m2 pair
+       array/float
+           Symmetric mass ratio for the m1, m2 pair
+    '''
+    m1, m2 = np.array(m1), np.array(m2)
+    return (m1*m2)**.6/(m1+m2)**.2, m1*m2/(m1+m2)**2  
+
+def Mmch_to_m1m2(M, mch):
+    '''Get component masses from chirp and total mass
+
+       Parameters
+       ----------
+       M: array/float
+           Total mass of the binary
+       mch: array/float
+           Chirp mass of the binary
+
+       Returns
+       -------
+       array/float
+           First mass of the binary
+       array/float
+           Second mass of the binary
+    '''
+    M, mch = np.array(M), np.array(mch)
+    b = (mch**5 * M)**(1./3)
+    m1 = (M - np.sqrt(M**2 - 4*b))/2 
+    return m1, M - m1
+
+def mcheta_to_m1m2(mch, eta):
+    '''Get component masses from chirp mass and mass ratio
+
+       Parameters
+       ----------
+       mch: array/float
+           Chirp mass of the binary
+       eta: array/float
+
+       Returns
+       -------
+       aarray/float
+           First mass of the binary
+       array/float
+           Second mass of the binary
+    '''    
+    mch, eta = np.array(mch), np.array(eta)
+    idx = np.where(eta > 0.24999)
+    
+    a, b = mch**2/eta**.2, mch/eta**.6
+    m2 = (b - np.sqrt(b**2 - 4*a))/2
+    m1 = a/m2
+    m1[idx] = m2[idx] = mch[idx]*(2**.2)
+        
+    return m1, m2
+
+def jacobian_m1m2_to_mcheta(mch, eta, minm1, maxm1):
```

Comment:
We have jacobians built into the distributions / transformations we also use for pycbc inference. Can you coordinate with @cdcapano  to use that functionality to avoid the duplication.

Reply:
I have removed the function. It was meant if the PDF of astro model was defined on the mch-eta space. Will bring it/Collin's version back if required in future.